Bottom: 590d78abca6568895476bf489103bd40362432d6
Top:    1a95d699dc4aadf0300d45e016e22739712affbe
Author: Ian Kent <raven@themaw.net>
Date:   2013-06-07 13:54:31 +0800

Refresh of autofs-5.0.7-fix-potential-null-dereference-in-lookup-ldap-lookup_mount.patch

---

diff --git a/modules/lookup_ldap.c b/modules/lookup_ldap.c
index 6e06e02..3371b3f 100644
--- a/modules/lookup_ldap.c
+++ b/modules/lookup_ldap.c
@@ -3012,7 +3012,7 @@ int lookup_mount(struct autofs_point *ap, const char *name, int name_len, void *
 				       mapent, ctxt->parse->context);
 	if (ret) {
 		time_t now = time(NULL);
-		int rv = CHE_OK;
+		int rv = CHE_FAIL;
 
 		/* Record the the mount fail in the cache */
 		cache_writelock(mc);
